content/applications/general.rst

```diff
@@ -13,3 +13,4 @@ Miscellaneous
     general/calendars
     general/in_app_purchase
     general/developer_mode
+    general/product_images
```

Comment:
If you're asking me, I would say @Feyensv 's suggestion is better: I think it would be better under Sales/Products etc.

A few months ago, I would have been in favor of a single "product configuration" category with all aspects of the products in there. (so, the opposite of my answer above :p )
But that's not what we are doing for this doc structure, especially after a long discussion with Fabien a while ago.
I think being consistent is more important and if "generic" aspects of products are left detailed in "Sales", that's where this doc should go.

if it goes in the "product" folder, it's not absolutely necessary to have the word "product" in the filename anymore. However, another keyword in the filename, related to the title (see next comment), could help improve the SEO.

Cheers :)

Reply:
I placed it under User Docs -> Sales -> Sales -> Product & prices. @AntoineVDV Should I make redirection for the link in the settings of Odoo, a PR to change this link in Odoo or both?

Reply:
Both because the URL is hard-coded in a stored view :grimacing: We'll remove the redirection in a few years.